jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -101,36 +101,37 @@ public ImageAndAuthorization call()
     EventHandlers eventHandlers = buildConfiguration.getEventHandlers();
     // Skip this step if this is a scratch image
     ImageConfiguration baseImageConfiguration = buildConfiguration.getBaseImageConfiguration();
-    if (baseImageConfiguration.getImage().isScratch()) {
+    ImageReference imageReference = baseImageConfiguration.getImage();
```

Comment:
I think we can combine these two lines? Doesn't look like `baseImageConfiguration` is used anywhere else.